website/source/docs/guides/leader-election.html.md

```diff
@@ -75,7 +76,7 @@ the lock.
 ### Watch the Session 
 
 All instances now remain in an idle waiting state. In this state, they watch for changes
-on key `dbservice`. This is because the lock may be released or the instance could fail, etc.
+on "mysql" key. This is because the lock may be released or the instance could fail, etc.
```

Comment:
Do you think it could cause confusion to call the key "mysql" here? Since it's `service/mysql/leader`.